src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -465,9 +465,11 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     CPU_SET_CALLBACK_COMMON(manager, nmsCallback, ov::pass::ConvertMulticlassNmsToMulticlassNmsIE);
     CPU_SET_CALLBACK_COMMON(manager, nmsCallback, ov::pass::ConvertMatrixNmsToMatrixNmsIE);
     CPU_SET_CALLBACK_X64(manager,
-        [](const_node_ptr &node) -> bool {
+        [this](const_node_ptr &node) -> bool {
             std::string errorMsg;
-            return node::ScaledDotProductAttention::isSupportedOperation(node, errorMsg);
+            // Matching the pattern is a little complicated, so we just check if there is any state nodes.
```

Comment:
Seems like the comment should be more meaningful. We should say that current SDPA impl is optimized only for LLM models, so we decompose it for others to avoid perf regression.

Reply:
Got it, updated.